ert_shared/ensemble_evaluator/prefect_ensemble/client.py

```diff
@@ -30,7 +32,7 @@ async def _send(self, msg):
                     self.websocket = await self.get_websocket()
                 await self.websocket.send(msg)
                 return
-            except (ConnectionError, OSError):
+            except (ConnectionError, OSError, ConnectionClosedError):
```

Comment:
Is this a fix of some kind?

Reply:
Yes. Perhaps we should address this in a separate issue. What do you think @mortalisk.

Reply:
At least separate it out into a separate commit. But, the preferable option is to make it into a separate PR and get it in faster :)